README.md

```diff
@@ -1,44 +1,134 @@
+
+The fork supports Python 3, increases test coverage, set up CI and linters, maintains releases to [PyPI](https://pypi.org/project/galgebra/#history), improves [docs](http://galgebra.readthedocs.io) and has many bug fixes.
+
+Features
+--------------------
+
+### Geometric Algebra
+
+- Arbitrary Vector Basis and Metric
+- Scalar, Vector, Bivector, Multivector, Pseudoscalar, Spinor, Blade
+- Basic Geometic Algebra Operations
+  - Sum Difference
+  - Geometric Product
+  - Outer and Inner Products
+  - Left and Right Contractions
+  - Reverse, Dual, Exponential
+  - Commutator
+  - Projection, Reflection, Rotation
+  - Reciprocal Frames
+- Inspecting Base/Blade Representation
+- Symbolic Manipulations
+  - `expand`, `factor`, `simplify`, `subs`, `trigsimp` etc.
+
+Overloaded Python operators for basic GA operations:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/basic_op.svg?sanitize=true)
+
+### Geometric Calculus
 
-Installing Galgebra
+- Geometric Derivative
+- Submanifolds
+- Linear Transformations
+- Differential Operators
+
+The various derivatives of a multivector function is accomplished by multiplying the gradient operator vector with the function:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad.svg?sanitize=true) ![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad_cmp.svg?sanitize=true)
+
+Tip: an example for getting `grad` and `rgrad`:
+
+```python
+(ex, ey, ez, grad, rgrad) = 
+  MV.setup('e*x|y|z', metric='[1,1,1]', coords=symbols('x y z'))
+```
+
+### Printing
+
+- Enhanced Console Printing
+- Latex Printing
+  - out-of-the-box support for Jupyter Notebook
+  - PDF generation and croping support if you have `pdflatex`/`pdfcrop` installed
+
+Getting Started
 ---------------------
 
-### Dependencies
+After installing GAlgebra (see Section [Installing GAlgebra](#installing-galgebra) below), in a Jupyter Notebook:
+
+```python
+from sympy import symbols
+from galgebra.ga import Ga
 
-- [Python](https://www.python.org/) 2.7 & 3
-  - tests pass under both Python 2.7 & 3.6
-- [SymPy](https://www.sympy.org)
+from galgebra.printer import Format
+Format(Fmode = False, Dmode = True)
 
-### Installing Galgebra From PyPI
+st4coords = (t,x,y,z) = symbols('t x y z', real=True)
+st4 = Ga('e_t e_x e_y e_z',
+         g=[1,-1,-1,-1],
+         coords=st4coords)
+
+M = st4.mv('M','mv',f = True)
+
+M.grade(3).Fmt(3,r'\langle \mathbf{M} \rangle _3')
+```
+
+You will see:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/st4_M3.svg?sanitize=true)
+
+For detailed documentation and more examples, please check out https://galgebra.readthedocs.io/ .
+
+**NOTE:** If you are coming from [sympy.galgebra](https://docs.sympy.org/0.7.6.1/modules/galgebra/) or [brombo/galgebra](https://github.com/brombo/galgebra), please check out Section [Migration Guide](#migration-guide).
```

Comment:
I'd have gone with "please check out the Migration Guide section" instead, which IMO feels more natural in English.

Reply:
IMO the verbosity of "the xxx section" here is not necessary.

Cross references like "Section 1.2.3", "section 1.2.3", "section Dummy Section Name" is commonly used in academic writing. For example, in APA Style, sections are not numbered thus referenced by section names and there're actually LaTeX packages for this, see [1](https://texfaq.org/FAQ-nameref), [2](https://tex.stackexchange.com/a/317311/75671), [3](https://tex.stackexchange.com/a/121871/75671).

On Internet, it's most common to even skip the "section" prefix, because the hylink speaks for itself. But there's one tiny issue for galagebra:

The same README.md is used both on Github and on readthedocs and the latter has a different mechanism to name the section anchor, so the hyperlink in README.md currently only works on Github, the users would be confused on readthedocs. That's why I added the "section" prefix to prompt the user to look for the corresponding section until I have a solution for both scenarios.

Now it seems to me that using "Section" instead of "section" is a little bit unnecessary so I'll change it back to "section".

Reply:
Makes sense; agreed on all points 👍